packages/core/package.json

```diff
@@ -492,7 +492,8 @@
         "umd-compat-loader": "^2.1.2",
         "vue-loader": "^17.2.2",
         "vue-style-loader": "^4.1.3",
-        "webfont": "^11.2.26"
+        "webfont": "^11.2.26",
+        "shlex": "^2.1.2"
```

Comment:
how does adding this package change the package-lock by 5405 😨. Also can you summarize what this package is and why we need it?

Reply:
It's only 8 lines now after rebasing.

After discussion with @justinmk3, this package is needed for parsing shell commands.